### ISSUE TITLE
TypeScript error when specifying "fields" in params

### ISSUE THREAD

**Issue by chowlett** [sensedeep/dynamodb-onetable#33]
The specific error is "error TS2322: Type '{ index: string; parse: true; reverse: true; fields: string[]; }' is not assignable to type 'OneParams'. Object literal may only specify known properties, and 'fields' does not exist in type 'OneParams'". Reproducible on 1.3.6.

Maybe consider the following as a possible fix? (worked for me on 1.3.6)

```
--- a/src/Model.d.ts
+++ b/src/Model.d.ts
@@ -130,6 +130,7 @@ export type OneParams = {
     delete?: object,
     execute?: boolean,
     exists?: boolean,
+    fields?: string[]
     hidden?: boolean,
     index?: string,
     limit?: number,
```

**Comment by mobsense** [comment]
Thank you. That looks correct.

**Comment by mobsense** [comment]
Fixed in 1.3.7
